src/libsystemd-network/sd-dhcp-server-lease.c

```diff
@@ -215,7 +215,8 @@ static int dhcp_server_lease_append_json(sd_dhcp_server_lease *lease, JsonVarian
                                  JSON_BUILD_PAIR_BYTE_ARRAY("ClientId", lease->client_id.raw, lease->client_id.size),
                                  JSON_BUILD_PAIR_IN4_ADDR_NON_NULL("Address", &(struct in_addr) { .s_addr = lease->address }),
                                  JSON_BUILD_PAIR_STRING_NON_EMPTY("Hostname", lease->hostname),
-                                 JSON_BUILD_PAIR_FINITE_USEC("ExpirationUSec", lease->expiration)));
+                                 JSON_BUILD_PAIR_FINITE_USEC("ExpirationUSec", lease->expiration.boottime),
```

Comment:
ithink it would make sense to store the boot id here too, i.e. sd_id128_get_boot(), after all without it it's very hard to make sense of the boottime clock. but if you store the boot id as well, then it's very easy for people to figure out if they can use the boot time, or have to use the wallclock timestamp